packages/toolpad-core/src/internals/demo.tsx

```diff
@@ -1,6 +1,9 @@
 import * as React from 'react';
+import { DocsContext as Docs } from '../shared/context';
```

Comment:
I've made a few small changes to the internals too here https://github.com/mui/toolpad/pull/4268, also just trying to fix that we had both `internal` and `internals` folders. Let me know if you agree with those.

It's just these 2 commits: https://github.com/mui/toolpad/pull/4268/commits/9896d0d6e1f308dad0c5fec15c92295e59377db9 and https://github.com/mui/toolpad/pull/4268/commits/6dfee2f38f8a0526b6ebed24ee1b007deed0e038

Reply:
Got it, I'll pull those into this PR